IS7024Project/Pages/Index.cshtml

```diff
@@ -34,6 +34,7 @@
             top: 200px;
             left: 0;
             width: 100%;
+            padding-top: 250px
```

Comment:
Used "padding-top" function instead of
since I believe it gives more flexibility.

Reply:
Definitely... a good idea.  Rationale: It's easy to have different CSS stylesheets for different media types (screen, mobile, print), so the padding can be tailored to that display.  On the other hand, a br tag is there always.... thus, it's less flexible.  Always use padding/margin CSS styles; avoid multiple br tags.